src/cpp/session/modules/SessionCodeTools.R

```diff
@@ -176,6 +176,8 @@
 
    code <- deparse(func, width.cutoff = 59, control = control)
    
+   Encoding(code) <- "UTF-8"
```

Comment:
Are we sure that the function body is deparsed using UTF-8 encoding? It seems like this may not be true on Windows (or perhaps it depends on the value of the `getOption("encoding")` R option)

Reply:
I agree. This may not be a safe change. @XiangyunHuang Could you check if the original issue https://d.cosx.org/d/419665 could be fixed without this change? In general, we tend to prefer minimal changes that fix a problem.